views/session.hbs

```diff
@@ -0,0 +1,54 @@
+<div class="spacing-32">
+
+  <div class="spacing-32">
+    {{{typography "Session Information" "headline37" "foregroundDefault"}}}
+
+    {{{typography
+      "Your browser holds an active Ory Session for {{ slug }} and you are currently logged in as {{ identity.traits.email }} - changing properties inside Acount Settings will be reflected in the decoded Ory Session."
+      "small"
+      "foregroundMuted"
+    }}}
+  </div>
+
+  <div class="spacing-32-horizontal wrap-content">
+     {{{sessionCodeBox}}}
+    <div class="spacing-8 traits-box">
+      {{#each traits}}
+        <div class="spacing-8">
+          {{{typography @key "caption" "foregroundDefault" "bold"}}}
+          {{{typography this "caption" "foregroundDefault"}}}
+          {{{divider true "traits-box-divider"}}}
+        </div>
+      {{/each}}
+    </div>
+  </div>
+  
+  {{{divider false "divider-left"}}}
+
+<script type="application/javascript">
+  // Get the current breakpoint
+  const getBreakpoint = function () {
+    return window.getComputedStyle(document.body, ':before').content.replace(/\"/g, '');
+  };
+
+  const setSessionCodeBoxCollapse = function () {
+    breakpoint = getBreakpoint();
+    const codeBoxInput = document.getElementsByClassName("session-code-box")[0].getElementsByTagName("input")[0]
```

Comment:
Could we move this into the `CodeBox` element? I think this should be possible with pure CSS as well, if we refactor the CodeBox a bit.

Reply:
I don't think you can do this with only css since you cannot change the state (input element is checked) of the collapse based on the screen size in css.

Since how this works is you have a checkbox hidden and a label (icon) linked to the checkbox. Since the checkbox is hidden the label is clickable and sets the state (checked or not checked). CSS can react to an input box state but not change it.

Reply:
>I don't think you can do this with only css since you cannot change the state (input element is checked) of the collapse based on the screen size in css.

I'd have to think about it. I really don't like doing these kinds of things on load, and I think it should be possible to have the interactivity just in CSS. It might be more hacky than this, though. So not sure if that'd be a win. :D 

The `breakpoint` variable should be `const` or `let`. Just noticed.

Reply:
Hmm, yeah i would also need to think about it a bit. But this was the best solution I could come up with for the collapse on breakpoint without breaking the codebox entirely.

Reply:
Sounds good, for now, then.